mod/nodeinfo.php

```diff
@@ -185,10 +185,10 @@ function nodeinfo_cron() {
 	}
         logger("cron_start");
 
-	$users = q("SELECT `user`.`uid`, `user`.`login_date`,
-			(SELECT MAX(`changed`) FROM `item` FORCE INDEX (`uid_wall_changed`) WHERE `wall` AND `uid` = `user`.`uid`) AS `lastitem_date`
+	$users = qu("SELECT `user`.`uid`, `user`.`login_date`, `contact`.`last-item`
 			FROM `user`
```

Comment:
[standards] Please fix SQL query indentation (two tabs instead of one expected) 😊